hibernate-types-43/src/test/java/com/vladmihalcea/hibernate/type/json/PostgreSQLJsonBinaryTypeCustomJsonSerializerConfigurationTest.java

```diff
@@ -75,8 +76,10 @@ public Void apply(EntityManager entityManager) {
             @Override
             public Void apply(EntityManager entityManager) {
                 Event event = entityManager.find(Event.class, 1L);
-                assertEquals("2.25262562526626", event.getLocation().getReference().toString());
 
+                BigDecimal actualToCompare = event.getLocation().getReference().setScale(2, BigDecimal.ROUND_HALF_UP);
```

Comment:
This change shouldn't be required for this Pull Request.

Reply:
These tests are broken on my machine
https://i.imgur.com/YSsBHLI.png

Reply:
That's weird because it works just fine for me